docker-compose.yml

```diff
@@ -8,9 +8,6 @@ services:
         - NEW_USER=${USER}
     container_name: rally-tests
     volumes:
-      - $HOME/.gitconfig:/home/${USER}/.gitconfig
-      - $HOME/.github:/home/${USER}/.github
-      - $HOME/.gnupg:/home/${USER}/.gnupg
       - ${PWD}/.:/home/${USER}/rally
```

Comment:
I missed this in my first pass. Doesn't this assume that the user's Rally project directory is available in `~/rally`?

Reply:
Correct. The CWD is changed here: https://github.com/dliappis/rally/blob/79c6ab8fad02f6509bb510c116c8d3de8b6b0cc0/docker-compose.yml#L14 .

The reason it's done this way is that any dot files created under `/home/$USER` inside the docker container shouldn't interfere with the dot files (e.g. `~/.rally`) of the host system.

Reply:
Ah, you're right. This is the path in the container, not on the host.